components/movementsensor/gpsutils/vrs.go

```diff
@@ -4,12 +4,9 @@ package gpsutils
 import (
```

Comment:
the file could be used outside of gpsrtk, if a user chose to implement their own version of rtk. the same can be said about ntrip.go imo, and since vrs.go is intended to work with the structs from ntrip.go I believe they should stay in the same package.

I can also see an argument for moving the vrs code into ntrip.go, or at least making the functions into methods on the ntripInfo/Sourcetable structs that we defined